.github/pull_request_template.md

```diff
@@ -1,45 +1,19 @@
-<!--
-Thank you for submitting a PR to the rust rpm implementation!
--->
-
-# PR Content Desc
-
-<!---
-DELETE all that do not apply:
--->
-
-- 🩹 Bug Fix
-- 🦚 Feature
-- 📙 Documentation
-- 🦣 Legacy
-- 🪣 Misc
```

Comment:
Tags would be better for this I think

Reply:
Tags cannot be added by external contributors

Reply:
What is the value add of these groups though?  If the goal is to make the contributor think about the changes they are making, then changelog entries already largely accomplish this in a more effective (IMO) way.

Reply:
I see your point, but they all apply to changelogs as well.

Reply:
Well, the value in that case is mostly for the end consumers not the maintainers, but it's there for someone  :)